docs/framework/guides/creating-simple-plugin.md

```diff
@@ -295,4 +295,6 @@ If you would like to read more tutorials, check out the following one:
 * {@link framework/guides/tutorials/implementing-a-block-widget Implementing a block widget}
 * {@link framework/guides/tutorials/implementing-an-inline-widget Implementing an inline widget}
 
-If you are more into reading about the CKEditor 5 architecture, check out the {@link framework/guides/architecture/intro Introduction to CKEditor 5 architecture}.
+If you want to read more about the CKEditor 5 architecture, check out the {@link framework/guides/architecture/intro Introduction to CKEditor 5 architecture}.
+
+If you would like know, how to check what kind of HTML output our plugins return or what UI components are exposed by them, check out the {@link framework/guides/contributing/plugin-metadata plugin metadata} guide.
```

Comment:
This will be rather done in this guide - #9414.

In this guide (creating a simple plugin) we should mention here that all 3rd-party CKEditor 5 packages can document their plugins using this meta file. This will allow for easier integrations with their plugins.

Reply:
I rewrote this sentence - I hope it's better now.